### PR TITLE
Fix: broken link to PHP page about error_reporting

### PR DESCRIPTION
**Pull request by spekulatius**
Hey @kylekatarnls 

while looking around I've found a broken link in the docs. Here is a fix for it.

Peter

### CODE REVIEW

File: doc/en/options.md [hunk 1]
```diff
@@ -452,15 +452,10 @@ the parser).
...
 
-Allow to handle PHP errors display that comes up during the
-template execution. By default, errors raised by current
-PHP setting (see
-[error_reporting](php.net/manual/en/function.error-reporting))
-are turned into exception Phug is able to trace the origine
-in the template pug code. Other errors are hidden.
+Allow to handle PHP errors display that comes up during the template execution. By default, errors raised by current PHP setting (see [error_reporting](https://php.net/manual/en/function.error-reporting)) are turned into exception Phug is able to trace the origine in the template pug code. Other errors are hidden.
+
+You can pass an custom error level that will override the PHP setting.
```
Comment (kylekatarnls) [inline doc/en/options.md]: Keep multiline wrapping (easy reading of the MD raw file in a small window) + fix "origine" to "origin" typo as we're there.
```suggestion
Allow to handle PHP errors display that comes up during
the template execution. By default, errors raised by current
PHP setting (see
[error_reporting](https://php.net/manual/en/function.error-reporting))
are turned into exception Phug is able to trace the origin
in the template pug code. Other errors are hidden.

You can pass an custom error level that will override
the PHP setting.
```

**Review by kylekatarnls — CHANGES_REQUESTED** [review]
Thank you. Please keep the wrapping consistent with the rest of the options.md file (line length < ~70).

Note: you also can "Allow edition by maintainer" so I can directly complete and merge in this PR.

**Comment by spekulatius** [comment]
I've committed the change @kylekatarnls. Not sure how to 'Allow edition by maintainer' (after opening it?)

**Review by kylekatarnls — APPROVED** [review]

**Comment by spekulatius** [comment]
Let me know if anything else needs changes @kylekatarnls

**Pull request merged**